src/React/README.md

```diff
@@ -0,0 +1,59 @@
+# Symfony UX React
+
+Symfony UX React integrates [React](https://reactjs.org/) into Symfony applications. It provides tools 
+to render React components from Twig.
+
+**This repository is a READ-ONLY sub-tree split**. See
+https://github.com/symfony/ux to create issues or submit pull requests.
+
+## Usage
+
+```bash
+composer require symfony/ux-react
```

Comment:
```
$ composer require symfony/ux-react

# Don't forget to install the JavaScript dependencies as well and compile
$ yarn install --force
$ yarn encore dev
```

(though, unrelated, I'm thinking we should start leaning towards showing `npm` commands first, as the user will definitely have this installed).